File: app/src/main/java/org/wit/emergencyescape/main/MainApp.java
package org.wit.emergencyescape.main;

import java.util.ArrayList;
import java.util.List;
import android.app.Application;
import android.util.Log;
import org.wit.emergencyescape.models.LocationModel;

public class MainApp extends Application
{
    public List <LocationModel>  locations = new ArrayList<>();
    @Override
    public void onCreate()
    {
        super.onCreate();
        Log.v("Escaper", "Landmark App Started");
    }
}

File: app/src/main/java/org/wit/emergencyescape/main/MainApp.kt
package org.wit.emergencyescape.main

import android.app.Application
import com.google.firebase.FirebaseApp
import com.google.firebase.auth.FirebaseAuth
import org.jetbrains.anko.AnkoLogger
import org.jetbrains.anko.info
import org.wit.emergencyescape.models.*

class MainApp : Application(), AnkoLogger {

    lateinit var locations : ArrayList<LocationModel>
    lateinit var firestore : LocationFireStore
    override fun onCreate() {
        super.onCreate()
        firestore = LocationFireStore(applicationContext)
        locations = firestore.locations

        info("Hillfort started")

        //
    }
}